### ISSUE TITLE
Menu button is not a button

### ISSUE THREAD

**Issue by xi** [django/djangoproject.com#1234]
The menu button for the mobile navigation is not accessible by keyboard because it is a `<div>` instead of a `<button>`

**Comment by pauloxnet** [comment]
How can your reporting of yours be verified in concrete? How would you change the button?

**Comment by RyanWalker277** [comment]
@pauloxnet The menu button is indeed a div , yet it works perfectly , I faced no issue as the button not being clickable .I tested the website on multiple mobile browsers but didn't find any such problem.Attaching screenshot for refrence.


![image](https://user-images.githubusercontent.com/32684077/196188942-39737369-6442-4898-9cac-9790a3e6875d.png)

**Comment by xi** [comment]
Let me be more specific about the steps to reproduce the issue:

Imagine you are a user with bad sight and bad hand-eye coordination. So you zoom in 400% and use the keyboard to navigate.

Expected experience: I can use the tab key to get to the menu button, and use the space key to open the menu.

What happens instead: The menu button is skipped. Instead the focus moves directly to the (invisible) menu.

**Comment by amit9838** [comment]
Fixed,  see PR  #1306

**Comment by aviralshastri** [comment]
![● mobile-menu js - djangoproject com - Visual Studio Code 24-01-2023 11_47_43](https://user-images.githubusercontent.com/104085688/214224842-3869865e-40c3-45a2-89b8-87ee2b71a1af.jpg)

**Comment by amit9838** [comment]
Ok so, do you want me to cancel my PR? right?

**Comment by Charan-Deep** [comment]
Please Assign me this issue.

**Comment by YashGodwal** [comment]
![Screenshot (75)](https://user-images.githubusercontent.com/96742430/229338314-3e18ab13-63d1-443c-964f-6a973f68187e.png)
A "menu button" can refer to a specific type of button that is used to access a navigation menu on a website. This type of button is typically represented by an icon with three horizontal lines, often referred to as a "hamburger menu" icon.

So, in this context, the statement "menu button is button in website" would be correct. A menu button on a website is a type of button that is used to access a navigation menu, and it is an important element of website design and user experience.

**Comment by kaushal-malpure** [comment]
@pauloxnet Can I also proceed with this issue?

**Comment by pauloxnet** [comment]
@kaushal-malpure feel free to work on this, no need to be assigned to work on an issue :)

**Comment by kaushal-malpure** [comment]
Hey @pauloxnet , 
As per issue raised menu button div tag changed to button tag (PR#1402).
Below is the attached screenshot for the same.
<img width="960" alt="Screenshot 2023-08-25 221958" src="https://github.com/django/djangoproject.com/assets/71086178/32ad6850-ac20-435f-b3c6-68fc66136ba5">

**Comment by pauloxnet** [comment]
@kaushal-malpure please answer questions in your PR
